src/software/thunderscope/widget_setup_functions.py

```diff
@@ -135,6 +137,7 @@ def setup_gl_widget(
     gl_widget.add_layer(validation_layer)
     gl_widget.add_layer(trail_layer, False)
     gl_widget.add_layer(debug_shapes_layer, True)
+    gl_widget.add_layer(max_dribble_layer, False)
```

Comment:
I don't mind making this on by default but let's see if others have strong opinions on this. I don't mind either way.